library/Freckle/App/Test.hs

```diff
@@ -122,8 +126,11 @@ instance HasLogger app => Example (AppExample app a) where
 instance HasTracer app => MonadTracer (AppExample app) where
   getTracer = view tracerL
 
-instance XRay.MonadTracer (AppExample app) where
-  getVaultData = pure Nothing
```

Comment:
You'll want to keep this. Anything still opting into `XRay` can function even with your changes, they just have to use `Database.XRay.runDB` and can't get the `MonadSql*` goodies. Though I know it practically doesn't matter, as we're really the only user, I see no real downside to not to saving removing this for when we remove everything else, in a breaking-change version bump.